ui/appui-react/src/appui-react/backstage/BackstageManager.tsx

```diff
@@ -7,71 +7,51 @@
  */
 
 import * as React from "react";
-import { BeEvent } from "@itwin/core-bentley";
 import { IconSpec } from "@itwin/core-react";
-import { CommandItemDef } from "../shared/CommandItemDef";
 import { UiFramework } from "../UiFramework";
-import { Backstage } from "./Backstage";
-
-/** Arguments of [[BackstageManager.onToggled]].
- * @public
- */
-export interface BackstageToggledArgs {
-  readonly isOpen: boolean;
-}
+import { BackstageToggledArgs, FrameworkBackstage } from "../framework/FrameworkBackstage";
+import { InternalBackstageManager } from "./InternalBackstageManager";
 
 /** Controls backstage.
  * @public
  */
 export class BackstageManager {
-  private _isOpen = false;
+  private internal = new InternalBackstageManager();
 
   /** Event raised when backstage is opened or closed. */
-  public readonly onToggled = new BeEvent<(args: BackstageToggledArgs) => void>();
+  public get onToggled() { return this.internal.onToggled; }
 
   public get isOpen() {
-    return this._isOpen;
-  }
-
-  private setIsOpen(isOpen: boolean) {
-    if (isOpen === this._isOpen)
-      return;
-    this._isOpen = isOpen;
-    this.onToggled.raiseEvent({
-      isOpen,
-    });
-    Backstage.onBackstageEvent.emit({ isVisible: isOpen }); // eslint-disable-line deprecation/deprecation
+    return this.internal.isOpen;
   }
 
   public open() {
-    this.setIsOpen(true);
+    return this.internal.open();
   }
 
   public close() {
-    this.setIsOpen(false);
+    return this.internal.close();
   }
 
   public toggle() {
-    this.setIsOpen(!this.isOpen);
+    return this.internal.toggle();
+  }
+
+  public getBackstageToggleCommand(overrideIconSpec?: IconSpec) {
+    return this.internal.getBackstageToggleCommand(overrideIconSpec);
   }
 
-  /** Get CommandItemDef that will toggle display of Backstage and allow iconSpec to be overridden */
+  /** Get CommandItemDef that will toggle display of Backstage and allow iconSpec to be overridden
+  */
   public static getBackstageToggleCommand(overrideIconSpec?: IconSpec) {
-    return new CommandItemDef({
-      commandId: "UiFramework.openBackstage",
-      iconSpec: overrideIconSpec ? overrideIconSpec : "icon-home",
-      labelKey: "UiFramework:commands.openBackstage",
-      execute: () => {
-        UiFramework.backstageManager.toggle();
-      },
-    });
+    return UiFramework.backstage.getBackstageToggleCommand(overrideIconSpec);
   }
 }
 
 /** Hook that returns isOpen flag of the backstage.
  * @public
  */
-export const useIsBackstageOpen = (manager: BackstageManager) => {
+export const useIsBackstageOpen = (manager: FrameworkBackstage) => {
```

Comment:
This is a breaking change?

Reply:
The interface is the same, it shouldnt be breaking.